extensions/binding/org.eclipse.smarthome.binding.mqtt/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,140 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="mqtt" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<bridge-type id="broker">
+		<label>MQTT Broker</label>
+		<description></description>
+
+		<config-description>
+			<parameter name="host" type="text" required="true">
+				<label>Broker Hostname/IP</label>
+				<description>The IP/Hostname of the MQTT broker in the form of hostname:port. The port is optional, if none is provided, the typical ports 1883 and 8883 (SSL) are used.</description>
```

Comment:
Ports must not be specified in here - a "network-address" is only hostname or IP address. A port should be a separate, optional config parameter.